R/bmi.R

```diff
@@ -1,107 +1,123 @@
-#' @title ifelse2
-#' 
+#' @title if_else2
+#'
 #' @description Custom ifelse function that evaluates missing (NA) values
-#' 
-#' @details If the logical argument (x) compares to a value that is `NA`, it is set to `FALSE`  
-#' 
+#'
+#' @details If the logical argument (x) compares to a value that is `NA`, it is
+#'  set to `FALSE`
+#'
 #' @param x A logical argument
-#' 
+#'
 #' @param a value if `x` is `TRUE`
-#' 
+#'
 #' @param b value if `x` is `FALSE`
-#' 
+#'
 #' @return a or b based on the evaluation of x
-#' 
-#' @examples 
+#'
+#' @examples
 #' age <- 12
-#' status <- ifelse2((age<18),"child", "invalid age")
+#' status <- if_else2((age < 18), "child", "invalid age")
 #' print(status)
-#' 
+#'
 #' age <- NA
-#' status <- ifelse2((age<18), "child", "invalid age")
+#' status <- if_else2((age < 18), "child", "invalid age")
 #' print(status)
-#' 
 #' @export
-ifelse2 <- function(x, a, b) {
+if_else2 <- function(x, a, b) {
```

Comment:
Should be moved to another module, `if_utils` since this is being used by age_cat_func also